MAUI/Button/Visual-States.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Visual states in .NET MAUI Button Control | Syncfusion
+description: Learn here about visual states support in the Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
```

Comment:
description: Learn about visual states support in the Syncfusion .NET MAUI Button (SfButton) control, its elements, and more.